framework/src/test/java/org/tron/core/net/messagehandler/SyncBlockChainMsgHandlerTest.java

```diff
@@ -64,6 +65,21 @@ public void testProcessMessage() throws Exception {
     method.setAccessible(true);
     boolean f = (boolean)method.invoke(handler, peer, message);
     Assert.assertTrue(!f);
+
+    Method method1 = handler.getClass().getDeclaredMethod(
```

Comment:
Is it possible to reproduce this scenario without using reflection?

Reply:
Okay, further optimization.